<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QMainWindow" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Connect Tracker</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_rec">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Start Tracking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3.Stop Tracking</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tx">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ty">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Ty:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tz">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Tz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Terror">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_valx">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_valy">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_valz">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vale">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <zorder>pushButton_connect</zorder>
   <zorder>textEdit_rec</zorder>
   <zorder>pushButton_stop</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>label_Tx</zorder>
   <zorder>label_Ty</zorder>
   <zorder>label_Tz</zorder>
   <zorder>label_Terror</zorder>
   <zorder>label_valx</zorder>
   <zorder>label_valy</zorder>
   <zorder>label_valz</zorder>
   <zorder>label_vale</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
